src/components/views/elements/AppTile.js

```diff
@@ -333,6 +359,7 @@ export default React.createClass({
                             alt={_t('Edit')}
                             title={_t('Edit')}
                             onClick={this._onEditClick}
+                            onLoad={this._onLoaded}
```

Comment:
This seems like a bizarre thing to be doing. I don't think this image's rendering has anything to do with the loading state of scalar.

Reply:
Meh... you're quite right, this should be on the iframe. Thanks for catching that!